README.md

```diff
@@ -1,170 +1,60 @@
 # nvim-lspconfig
 
-Collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
-The configurations are supported on a best-effort basis, and rely on contributions 
-from regular users to stay up-to-date.
-
-This repo handles automatically launching, initializing, and configuring language servers
-that are installed on your system.
+A collection of common configurations for Neovim's built-in [language server client](https://neovim.io/doc/user/lsp.html).
+This repo handles automatically launching and initializing language servers that are installed on your system.
 
 ## Install
 
-* Requires [Neovim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease). The configs in this repo
-  assume that you are using the latest [Neovim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly). 
-  Update Neovim and nvim-lspconfig before reporting an issue.
+* Requires [Neovim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease). Update Neovim and nvim-lspconfig before reporting an issue.
```

Comment:
I think the reduction to the "wall of text" here is awesome, as a lot of people seemingly don't realize that to get going, you can literally just install a server, setup this repo, and write a single line to your init script to have an insanely useful baseline. If that isn't plug-and-play, I'm not really sure what is.

One thing here to continue with the streamlining might be to add a table of contents @mjlbach? The README after this update serves as really useful overview, but people need to understand up front that this is not the end of the capabilities provided by this repository. The interface/baseline is really where the true power _starts_, so showing them how to get started, then providing resources to immediately change/add seems like it'd be useful.

Reply:
> The interface/baseline is really where the true power starts, so showing them how to get started, then providing resources to immediately change/add seems like it'd be useful.

What do you mean by this? The cut content will go into the wiki, so I was thinking of adding a mini-table of contents (sort of like what is there already) to this missing content.

Reply:
You've noted exactly what I was trying to say; the _README_ is a good "short read", but it is also important to show where people can learn more. My comment was mainly around the idea that people might not want to scroll to the bottom of this already short _README_, to do so, and presenting an early "catering to the users interest level" earlier on might serve better.

Reply:
This meaning pretty much a table of contents to the _README_ itself at the very top, one of the sections in that table of contents would maybe be a table to the Wiki to learn more, etc.

Reply:
Also, and I know you haven't gotten there yet, but this:

![image](https://user-images.githubusercontent.com/31262046/118424992-7bde3d80-b685-11eb-93c4-aa781c3283fb.png)

is a bit daunting (especially to people who have trouble moving their eyes slightly to the right ;) ). Not sure what other things _Github_ provides to spruce that up, but that might help as well.

Reply:
Sadly it doesn't provide anything. I tried deleting the HOME page but it results in a generic list of entries without the nice sidebar. I'm going to turn HOME into a table of contents like the coc wiki after moving content from the readme into the wiki.

Reply:
> add a table of contents

if you mean README's table of content, I don't think that's necessary as Github is now capable of generating TOC automatically and put it on a menu above the README. People just need to get used to it :p

![image](https://user-images.githubusercontent.com/51877647/118426371-4709cf00-b6f5-11eb-8002-3fdd8c0710fe.png)

Reply:
Ooh I didn't even know about that, looks nice!